additional.d.ts

```diff
@@ -1 +1,10 @@
 // declare here any additional module with no types
+
+declare module 'happo-cypress/task' {
```

Comment:
I should create a PR in DefinitelyTyped to include this types for Happo. I'll try to do this when I have more spare time.

Reply:
That's okay! I'll let you do that in your own time. 😉